apps/test/unit/lib/kits/maker/boards/FakeBoardTest.js

```diff
@@ -1,9 +1,18 @@
 import {itImplementsTheMakerBoardInterface} from './MakerBoardTest';
 import FakeBoard from '@cdo/apps/lib/kits/maker/boards/FakeBoard';
 import {expect} from '../../../../../util/reconfiguredChai';
+import {itMakesCircuitPlaygroundComponentsAvailable} from './circuitPlayground/CircuitPlaygroundBoardTest';
 
 describe('FakeBoard', () => {
-  itImplementsTheMakerBoardInterface(FakeBoard);
+  describe('Maker Board Interface', () => {
+    describe('implements shared Maker Board Interface', () => {
+      itImplementsTheMakerBoardInterface(FakeBoard);
+    });
+
+    describe('Circuit Playground components accessible from interpreter', () => {
+      itMakesCircuitPlaygroundComponentsAvailable(FakeBoard);
```

Comment:
The `itMakesCircuitPlaygroundComponentsAvailable` function creates its own describe block "Circuit Playground components accessible from interpreter" so you've got a double-nesting, here and later in CircuitPlaygroundBoardTest.  This doesn't break anything, but it has the odd side effect that the duplication shows up in your fully-qualified test names, like this:

> FakeBoard Maker Board Interface Circuit Playground components accessible from interpreter Circuit Playground components accessible from interpreter adds component constructors correct number of them

My own preference would be to leave the `describe` inside your helper function and remove the ones where you invoke the helper, especially since the function name says roughly the same thing.

The describe wrapped around `itImplementsTheMakerBoardInterface` is similarly redundant, although I see you're adding some context about it being a _shared_ interface.  That context might be cleaner as a comment than a describe block in this case.

Reply:
:laughing: "FakeBoard Maker Board Interface Circuit Playground components accessible from interpreter Circuit Playground components accessible from interpreter adds component constructors correct number of them"
^That's what the inside of my brain sounds like. Thanks for the good feedback.